src/less/waffle.less

```diff
@@ -91,3 +91,69 @@ th {
     }
   }
 }
+
+/**
+ * --------------------------------------------------------------------------*\
+ * Fixed header for scroll handling.                                         *\
+ * http://jsfiddle.net/drueter/yqgB5/16/                                     *\
+ * by https://github.com/DavidRueter                                         *\
+ * --------------------------------------------------------------------------*\
+ */
+// Set table elements display to block
+table.waffle-fixedheader,
```

Comment:
You can probably use less syntax to avoid repeating everywhere this css class, no ?